flink-yarn/src/test/resources/META-INF/services/org.junit.jupiter.api.extension.Extension

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+org.apache.flink.util.TestLoggerExtension
```

Comment:
This needs to be excluded from the test-jar since it otherwise leaks into downstream modules (specifically, flink-yarn-tests)

Reply:
you should also double-check what the test jar is used for; if some test base is used in downstream projects then it cannot be converted to junit5 yet.

Reply:
@zentol I checked the module related about `test-jar`, there are `flink-yarn-test` and `flink-end-to-end-tests`.
May I  change these modules in the same PR based the relationships ?

Reply:
no; we should first port the users of the test-jar (as much as possible), then migrate flink-yarn (along with whatever couldn't be ported previously in downstream modules).